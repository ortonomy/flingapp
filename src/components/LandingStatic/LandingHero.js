// basic react imports
import React from 'react';

// dependent components
import LandingButton from './LandingButton';

// styles
import styles from './LandingHero.module.css';

// copy
import { Hero as HeroText } from '../../assets/copy/Landing';

const LandingHero = (props) => {
  return (
    <div className={styles.LandingHero}>
      <div className={styles.LandingHeroLeft}>
        <h1>{HeroText.heroTitle1}</h1>
        <h1>{HeroText.heroTitle2}</h1>
        <h1>{HeroText.heroTitle3}</h1>
        <p>{HeroText.heroSubTitle1}</p>
        <div className={styles.LandingHeroCTA}>
          <LandingButton 
            text={HeroText.heroButtonText1}
            size='cta'
            color='blue'
            icon='fa-rocket'
          />
        </div>
      </div>
      <div className={styles.LandingHeroRight}>
      </div>
      
    </div>
  )
}

export default LandingHero;