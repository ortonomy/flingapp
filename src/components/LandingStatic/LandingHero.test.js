import React from 'react';
import LandingHero from './LandingHero';

it('renders without crashing', () => {
  const wrapper = shallow(<LandingHero />);
  expect(wrapper).toMatchSnapshot();
});
