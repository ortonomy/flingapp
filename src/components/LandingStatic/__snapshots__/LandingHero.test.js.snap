// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LandingHero />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className={undefined}
>
          <h1>
                    Freelancer
          </h1>
          <h1>
                    Management.
          </h1>
          <h1>
                    For free.
          </h1>
          <p>
                    Keep track of your freelancers' performance and identify the best resources for your next project.
          </p>
          <div
                    className={undefined}
          >
                    <withRouter()
                              color="blue"
                              icon="fa-rocket"
                              link="/register"
                              size="cta"
                              text=" Get started "
                    />
          </div>
</div>,
        <div
          className={undefined}
/>,
      ],
      "className": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1>
              Freelancer
</h1>,
            <h1>
              Management.
</h1>,
            <h1>
              For free.
</h1>,
            <p>
              Keep track of your freelancers' performance and identify the best resources for your next project.
</p>,
            <div
              className={undefined}
>
              <withRouter()
                            color="blue"
                            icon="fa-rocket"
                            link="/register"
                            size="cta"
                            text=" Get started "
              />
</div>,
          ],
          "className": undefined,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Freelancer",
            },
            "ref": null,
            "rendered": "Freelancer",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Management.",
            },
            "ref": null,
            "rendered": "Management.",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "For free.",
            },
            "ref": null,
            "rendered": "For free.",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Keep track of your freelancers' performance and identify the best resources for your next project.",
            },
            "ref": null,
            "rendered": "Keep track of your freelancers' performance and identify the best resources for your next project.",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <withRouter()
                color="blue"
                icon="fa-rocket"
                link="/register"
                size="cta"
                text=" Get started "
/>,
              "className": undefined,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "color": "blue",
                "icon": "fa-rocket",
                "link": "/register",
                "size": "cta",
                "text": " Get started ",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className={undefined}
>
            <h1>
                        Freelancer
            </h1>
            <h1>
                        Management.
            </h1>
            <h1>
                        For free.
            </h1>
            <p>
                        Keep track of your freelancers' performance and identify the best resources for your next project.
            </p>
            <div
                        className={undefined}
            >
                        <withRouter()
                                    color="blue"
                                    icon="fa-rocket"
                                    link="/register"
                                    size="cta"
                                    text=" Get started "
                        />
            </div>
</div>,
          <div
            className={undefined}
/>,
        ],
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1>
                Freelancer
</h1>,
              <h1>
                Management.
</h1>,
              <h1>
                For free.
</h1>,
              <p>
                Keep track of your freelancers' performance and identify the best resources for your next project.
</p>,
              <div
                className={undefined}
>
                <withRouter()
                                color="blue"
                                icon="fa-rocket"
                                link="/register"
                                size="cta"
                                text=" Get started "
                />
</div>,
            ],
            "className": undefined,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Freelancer",
              },
              "ref": null,
              "rendered": "Freelancer",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Management.",
              },
              "ref": null,
              "rendered": "Management.",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "For free.",
              },
              "ref": null,
              "rendered": "For free.",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Keep track of your freelancers' performance and identify the best resources for your next project.",
              },
              "ref": null,
              "rendered": "Keep track of your freelancers' performance and identify the best resources for your next project.",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <withRouter()
                  color="blue"
                  icon="fa-rocket"
                  link="/register"
                  size="cta"
                  text=" Get started "
/>,
                "className": undefined,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "color": "blue",
                  "icon": "fa-rocket",
                  "link": "/register",
                  "size": "cta",
                  "text": " Get started ",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
